cache-filter/src/lib.rs

```diff
@@ -3,8 +3,11 @@
 const VM_ID: &str = "my_vm_id";
 
 mod configuration;
+#[cfg(not(feature = "unique_callout"))]
+mod dummy;
 mod filter;
 mod log;
 mod rand;
+#[cfg(feature = "unique_callout")]
```

Comment:
Renaming `dummy` to `unique_callout_dummy` will help place these two together.

Reply:
Done